pkg/kudoctl/cmd/init/manager.go

```diff
@@ -209,6 +196,45 @@ func generateDeployment(opts Options) *appsv1.StatefulSet {
 		},
 	}
 
+	if !opts.DisableManager {
```

Comment:
I think we should validate somewhere in the CLI that both manager as well as reloader cannot be disabled otherwise this will be pod without containers (?)

Reply:
oh smart, yeah I'll fix this.